src/raylib.cc

```diff
@@ -629,8 +629,12 @@ PyObject* wait_for_next_task(PyObject* self, PyObject* args) {
   if (!PyArg_ParseTuple(args, "O&", &PyObjectToWorker, &worker)) {
     return NULL;
   }
-  Task* task = worker->receive_next_task();
-  return PyCapsule_New(static_cast<void*>(task), "task", NULL); // This task is owned by the C++ worker class, so we do not deallocate it.
+  if (std::unique_ptr<Task> task = worker->receive_next_task()) {
+    PyObject* pyobj = PyCapsule_New(task.get(), "task", TaskCapsule_Destructor);
```

Comment:
What do you think about writing `task.release()` here instead of `task.get()` and getting rid of `task.release()` in the code below?